src/components/Search/SearchAutocomplete/SearchAutocomplete.style.tsx

```diff
@@ -55,12 +55,14 @@ const MobileWrapperClosed = css`
   height: unset;
   width: ${props => props.theme.searchbar.mobileClosedWidth};
   padding: ${props => props.theme.searchbar.mobileClosedPadding};
-  margin-left: ${props => props.theme.searchbar.mobileClosedMarginLeft};
   max-width: ${props => props.theme.searchbar.mobileClosedMaxWidth};
+  margin: auto;
```

Comment:
oo any reason to edit these margins? nixing the margin-left and adding margin:auto throws off the spacing and width of the searchbar slightly

prod (search bar smaller and set to the right)-
<img width="377" alt="Screen Shot 2021-05-17 at 6 49 11 PM" src="https://user-images.githubusercontent.com/44076375/118565779-c96bb080-b740-11eb-891b-e447504f5909.png">

feature branch-
<img width="378" alt="Screen Shot 2021-05-17 at 6 49 20 PM" src="https://user-images.githubusercontent.com/44076375/118565766-c40e6600-b740-11eb-896f-6fdb33265b68.png">

Reply:
reverting this seemed to work, thanks for the callout.